backends/advanced/src/advanced_omi_backend/workers/speaker_jobs.py

```
@@ -15,6 +15,88 @@
 logger = logging.getLogger(__name__)
 
 
+def _merge_overlapping_speaker_segments(
+    segments: list[dict],
+    overlap: float
```

Ruff (0.14.11)

Unused function argument: `overlap`

(ARG001)